htdocs/societe/card.php

```diff
@@ -554,9 +554,8 @@
 			}
 			//var_dump($object->array_languages);exit;
 
-			if (GETPOST('deletephoto')) {
```

Comment:
You must keep the code in this if, used to delete the logo without uploading another one.

Reply:
I'm surprised, according to my tests, this part of code is not necessary.

Reply:
At 780, we delete only if $object->logo != '' but at line 557 (the code you mentioned above I've deleted) we remove de content of $object->logo. So the code at 780 is never executed.